src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -644,95 +575,71 @@ public enum DiamondPearlPromos implements LogicCardInfo {
             }
           }
           move "Metallic Bolt", {
-            text "60 damage. Energy attached to Magnezone. If you do, this attack�s base damage is 120 instead of 60."
-            energyCost L, M, C, C, L, M
+            text "60 damage. You may discard a [L] Energy and a [M] Energy attached to Magnezone. If you do, this attack's base damage is 120 instead of 60."
+            energyCost L, M, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 60
+
+              if (confirm(text)) {
+                damage 60
+
+                afterDamage {
+                  discardSelfEnergy L
```

Comment:
The energy that will be discarded should be chosen before damage happens. There is a static discardSelfEnergyAfterDamage that accomplishes this.